adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -315,6 +315,7 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 					opI.ClusterName)
 				opID := opI.ClusterName
 				opI.ClusterName = getClusterName(requestInterceptClustersNamePrefix, organizationID, vHost, apiTitle, apiVersion, opID)
+				operationalReqInterceptors[method] = opI // since cluster name is updated
```

Comment:
Isn't it better to make opI a pointer so the above will update the original clustername?